packages/firebase_admob/lib/firebase_admob.dart

```diff
@@ -82,6 +82,65 @@ class MobileAdTargetingInfo {
   }
 }
 
+// The types of ad sizes supported for banners.
+enum AdSizeType {
```

Comment:
It looks like indices of these enums are significant. That's worth a '//' implementation comment.

Reply:
Should I just add to the comment that's there and indicate how I'm using the index values? If there's an example of a similar comment, feel free to point me to it.

Reply:
https://docs.flutter.io/flutter/scheduler/SchedulerPhase-class.html uses an enum's index this way

It feels a little brittle to rely on the index because adding another id could disturb things.  On the other hand, you could easily write a regression test that checked this

Reply:
Makes sense. What about passing the String name of the value, then, rather than using the index?